app/views/vouchers/_form.haml

```diff
@@ -6,7 +6,11 @@
     = render 'voucher', f: f, v: v
   .d-flex
     - if vouchers.count > 0
-      = f.button :submit, value: 'Update Vouchers', class: 'btn btn-primary ml-auto', disabled: @subprogram.closed?
+      - show_confirm = Voucher.confirmation_text != Voucher.translated_confirmation_text
+      - submit_data = {}
+      - if show_confirm
+        - submit_data.merge!({ confirm: Voucher.translated_confirmation_text })
+      = f.button :submit, value: 'Update Vouchers', class: 'btn btn-primary ml-auto', disabled: @subprogram.closed?, data: submit_data
```

Comment:
I thought you were only going to display the confirmation if the user was making vouchers active?